### PR TITLE
Adding XZ decompression functionality for skiboot

### PR DESCRIPTION
**Pull request by e-liner**
We have already merged hostboot and open-power/pnor changes, this is the final change for RTC:125550.

This change will allow a user the option to XZ compress the skiboot image.  The BR2_TARGET_SKIBOOT_XZ config will need to be enabled for the specific configuration, and then the skiboot image will be compressed.  This change is only for compressing and decompressing the skiboot image section.  If another section needs compression, we would have to enable decompression inside hostboot for that specific section, and make additional pnor changes.

---
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/417)

### CODE REVIEW

File: openpower/package/openpower-pnor/Config.in [hunk 1]
```diff
@@ -60,3 +71,8 @@ config BR2_OPENPOWER_TARGETING_ECC_FILENAME
         string "Name of openpower binary targeting file"
         help
             String used to define name of openpower targeting binary file, ecc protected
+
+config XZ_COMPRESSION_ENABLED
```
Comment (williamspatrick) [inline openpower/package/openpower-pnor/Config.in]: Need a package scoped variable like 'BR2_OPENPOWER_PNOR_XZ_ENABLE'.

File: openpower/package/openpower-pnor/openpower-pnor.mk [hunk 2]
```diff
@@ -8,7 +8,7 @@
 # make doesn't care for quotes in the dependencies.
 XML_PACKAGE=$(subst $\",,$(BR2_OPENPOWER_XML_PACKAGE))
 
-OPENPOWER_PNOR_VERSION ?= 1e4410052a26acf4b439226860afae9507f0d94a
+OPENPOWER_PNOR_VERSION ?= f5b523fb0a7fe312c5d011e9853f53c22a2f2262
```
Comment (williamspatrick) [inline openpower/package/openpower-pnor/openpower-pnor.mk]: Minor, but a rebase should pick this version up already.  I believe the jenkins cronjob got it in last night.

File: openpower/package/openpower-pnor/openpower-pnor.mk [hunk 3]
```diff
@@ -24,6 +24,10 @@ endif
 
 endif
 
+ifeq ($(XZ_COMPRESSION_ENABLED),y)
+OPENPOWER_PNOR_DEPENDENCIES = host_xz
```
Comment (williamspatrick) [inline openpower/package/openpower-pnor/openpower-pnor.mk]: += , not =.

Comment (e-liner) [inline openpower/package/openpower-pnor/openpower-pnor.mk]: Done.

File: openpower/package/openpower-pnor/Config.in (hunk 1 above)
Comment (e-liner) [inline openpower/package/openpower-pnor/Config.in]: Done.

**Pull request closed without merging**